test/docfx.Test/lib/MonikerRangeParserTest.cs

```diff
@@ -126,6 +128,7 @@ public void TestEvaluateMonikerRange(string rangeString, string expectedMonikers
         [InlineData(">netcore<-1.0 ||| >netcore-2.0", "Expect a comparator set, but got `| >netcore-2.0`")]
         [InlineData(">netcore<-1.0 || ||", "Expect a comparator set, but got ` ||`")]
         [InlineData(">netcore<-1.0 || <", "Expect a moniker string, but got ``")]
+        [InlineData(">*", "Moniker `*` is not defined")]
```

Comment:
what if `* && >netcore-1.0` or `* || *`

Reply:
the only operator should be `=` as described

Reply:
Basically, only `*` is allowed

Reply:
so should `=*` also be disallowed?

Reply:
they are essentially the same though... `*` will be `=*` in our parser